vulnerability_report.md

```diff
@@ -34,10 +34,17 @@ Vulnerability - 6 Exposure:
 
 This site uses old, crummy, cracked encryption: MD5.  These passwords are crackable if someone really really want to - and because of the other vulnerabilities on the site (see A1, above), the poorly-hashed user passwords are discoverable.
 
-Repair: Use modern encryption for all sensitive data.
+Repair: Use modern encryption for all sensitive data.  I don't have sufficient security expertise to advise you on the optimal encryption choices for your use case, but AES looks pretty good.  
+
+See https://en.wikipedia.org/wiki/Advanced_Encryption_Standard
```

Comment:
The Django docs on security have recommendations about which password hashes to use. https://docs.djangoproject.com/en/1.9/topics/auth/passwords/